pkgs/by-name/ro/rockyou/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, fetchFromGitLab
+, stdenvNoCC
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "rockyou";
+  version = "unstable-2023-05-12";
+
+  src = fetchFromGitLab
+    {
+      group = "kalilinux";
+      owner = "packages";
+      repo = "wordlists";
+      rev = "f980044ab589841352cfd5e962caa1d0fe01e562";
+      hash = "sha256-s+k7iuAirz7lJCLvBdu0RQ9Nmcf5C0AOiwr1OMBf8xs=";
+    } + "/rockyou.txt.gz";
+
+  unpackCmd = ''
+    mkdir src
+    gzip -dc ${src} > src/rockyou.txt
+  '';
+
+  installPhase = ''
+    install -m 444 -D rockyou.txt $out/share/wordlists/rockyou.txt
+  '';
+
+  meta = with lib; {
+    description = "A famous wordlist often used for brute force attacks";
+    homepage = "https://www.kali.org/tools/wordlists/";
+    license = "unknown";
```

Comment:
I would be fine with adding creative the license that ieee uses to distribute the files, but then we might want to also pull the file from them and I don't if we can since it requires a account to download them.

Reply:
That makes sense. Maybe we should relicense it as MIT if we go with the `SecLists` repo, as it is marked as such.

Reply:
I changed it to cc-by-40 and added a comment.